seneca.js

```diff
@@ -721,23 +728,27 @@ function make_seneca (initial_options) {
     var pattern_rules = _.clone(action.validate || {})
     _.each(pattern, function (v, k) {
       if (_.isObject(v)) {
-        pattern_rules[k] = v
+        pattern_rules[k] = _.clone(v)
         delete pattern[k]
       }
     })
 
-    actmeta.rules = pattern_rules
-
     var addroute = true
 
+    // TODO: deprecate
```

Comment:
the term _args_ needs to be replaced by the term _msg_, as it better reflects the inherent distributed nature of the system
the fields in actmeta form an API, so we need to be careful with this transition